src/plugins/geomap/_base.scss

```diff
@@ -324,3 +315,19 @@ button {
 		z-index: 10000;
 	}
 }
+
+.geomap-aoi {
+	legend {
+		border: 0;
+		font-size: 1em;
+		margin-bottom: 1em;
+	}
+
+	button {
+		&.geomap-geoloc-aoi-btn {
+			position: absolute;
+			top: auto;
+			right: 15px;
```

Comment:
SCSS Lint is complaining about the order of property (it wants it in alphabetical order).

Please rewrite into:

```css
position: absolute;
right: 15px;
top: auto;
```